README.md

```diff
@@ -1,2 +1,5 @@
 # js-honeycomb-support
+
 Helpful stuff for adding honeycomb to a nodejs app
+
+# TODO: more docs
```

Comment:
I think it would be helpful to have at least an example of how you would use this lib here to start with.

Reply:
good point, I'll add that